tools/RELEASE.md

```diff
@@ -0,0 +1,40 @@
+# Release Puppet Extension for VSCode
+
+## Pre-release steps
+
+### Vendor Latest Puppet Language Server and Syntax Files
+
+1. Ensure [puppet-editor-services](https://github.com/lingua-pupuli/puppet-editor-services) has been tagged and release process followed. Note version.
```

Comment:
I think the numbering got outta whack here.

Reply:
Are you are referring to the `1.` on every line, if so that's a markdown convention. Or are you are referring to the order of vendor editor services then vscode?